.openshift-ci/crawler/setup-staging.sh

```diff
@@ -1,16 +1,25 @@
 # The below variables ontain a comma delimited list of GCP buckets,
-# Scripts may read from all buckets but only write to the first bucket in the list.
-KERNEL_PACKAGE_STAGING_BUCKET="gs://stackrox-kernel-packages-staging/copy"
-KERNEL_BUNDLE_STAGING_BUCKET="gs://collector-kernel-bundles-public"
-KERNEL_PACKAGE_BUCKET="${KERNEL_PACKAGE_STAGING_BUCKET}"
-KERNEL_BUNDLE_BUCKET="${KERNEL_BUNDLE_STAGING_BUCKET}"
-
-echo "KERNEL_BUNDLE_STAGING_BUCKET=${KERNEL_BUNDLE_STAGING_BUCKET}"
-echo "KERNEL_PACKAGE_STAGING_BUCKET=${KERNEL_PACKAGE_STAGING_BUCKET}"
+# Scripts may read from all buckets but only write to the *first bucket* in the list.
+
+KERNEL_PACKAGE_BUCKET="gs://stackrox-kernel-packages-staging/copy"
+KERNEL_BUNDLE_BUCKET="gs://collector-kernel-bundles-public"
+
+KERNEL_PACKAGE_STAGING_BUCKET="gs://stackrox-kernel-packages-staging/"
+KERNEL_BUNDLE_STAGING_BUCKET="gs://stackrox-kernel-bundles-staging/"
+
+if [[ "$BRANCH" =~ ^(master|main)$ ]]; then
+
+    echo "Using production buckets"
+    export KERNEL_PACKAGE_BUCKET="${KERNEL_PACKAGE_BUCKET}"
+    export KERNEL_BUNDLE_BUCKET="${KERNEL_BUNDLE_BUCKET}"
+
+else
+
+    echo "Using staging buckets"
+    export KERNEL_PACKAGE_BUCKET="${KERNEL_PACKAGE_STAGING_BUCKET},${KERNEL_PACKAGE_BUCKET}"
+    export KERNEL_BUNDLE_BUCKET="${KERNEL_BUNDLE_STAGING_BUCKET},${KERNEL_BUNDLE_BUCKET}"
+
```

Comment:
Why are you using both the staging bucket here and the regular bucket? Also why are you defining this as a string and then splitting it into an array? Why not just declare it as an array?

Reply:
That's how it was implemented originally before migration, and there are bits of crawler that actually depends on that. The point is explained in the commentary:

```
# The below variables ontain a comma delimited list of GCP buckets,
# Scripts may read from all buckets but only write to the *first bucket* in the list.
```

So, in case of synchronizing the data this approach allows to fetch cache from both staging (not complete) and prod (complete), but write the results only to the staging bucket (the first one). This makes everything faster at the end.